drivers/gles3/rasterizer_scene_gles3.cpp

```diff
@@ -3644,11 +3644,11 @@ void RasterizerSceneGLES3::_post_process(Environment *env, const CameraMatrix &p
 	}
 
 	if (!env || storage->frame.current_rt->flags[RasterizerStorage::RENDER_TARGET_TRANSPARENT] || storage->frame.current_rt->width < 4 || storage->frame.current_rt->height < 4) { //no post process on small render targets
-		//no environment or transparent render, simply return and convert to SRGB
+		//no environment or transparent render, simply return and convert to SRGB if KEEP_3D_LINEAR is not true.
```

Comment:
fix makes sense but i dont understand this comment, i think this was just something that was forgotten

Reply:
I thought it'd be good to document what the section currently does, but looking back at it the addition does seem like noise. The code is clear enough and extra verbosity isn't necessary.

Removed the change.